main/alsa-lib/template.py

```diff
@@ -16,7 +16,7 @@
 license = "LGPL-2.1-or-later"
 url = "https://www.alsa-project.org"
 source = f"{url}/files/pub/lib/{pkgname}-{pkgver}.tar.bz2"
-sha256 = "c86a45a846331b1b0aa6e6be100be2a7aef92efd405cf6bac7eef8174baa920e"
+sha256 = "9f3f2f69b995f9ad37359072fbc69a3a88bfba081fc83e9be30e14662795bb4d"
 options = ["linkundefver"]
```

Comment:
Builds for me fine without this line, or am I missing something?

Reply:
yeah seems that was fixed and i forgot about it

Reply:
thanks